static/js/components/EducationDialog.js

```diff
@@ -8,21 +9,22 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import { educationValidation } from '../util/validation';
 
 export default class EducationDialog extends ProfileFormFields {
-  constructor(props) {
+  constructor(props: {}) {
```

Comment:
I was thinking that `{}` might refer to an empty object specifically but I'm not sure. What's the difference between `Number` and `number`?